acceptance/tests/language/sensitive_data_type.rb

```diff
@@ -64,6 +64,15 @@
   sitepp_content = generate_manifest(test_resources)
   assertion_code = generate_assertions(test_resources)
 
+  refutation_resources = test_resources.collect do |assertion_group|
```

Comment:
strange.  it worked for me when using the same syslog files.  i'm not sure what you mean by "rest of the beaker log".
but i can see where it wouldn't work and am surprised it works in puppet pipeline. 

this is fine as is